tips/035_logit_bias.md

```diff
@@ -0,0 +1,269 @@
+## Effective AI Engineering #35: Logit Bias
+
+**Does your AI writing sound robotic with telltale patterns?** Every response contains em-dashes, starts with "Certainly!" or uses the same transitional phrases that scream "AI-generated content."
```

Comment:
This needs to be more narrative based. For example, really need to bring out the "everyone says it sounds like my writing looks like AI wrote it with all the em dashes... but that's because AI did write it. How can I make the writing appear more humanistic?" Well my dear, just avoid the tell-tale signs like em dashes with logit biases!